src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java

```diff
@@ -72,4 +92,44 @@ void should_copy_file_from_source_to_target_container() {
             .as("Envelope in the DB has been created")
             .hasValueSatisfying(envelope -> assertThat(envelope.status).isEqualTo(DISPATCHED));
     }
+
+    @Test
+    void should_not_process_blob_when_signature_verification_fails() throws Exception {
```

Comment:
imho this integration test is not needed, unit tests verify that the dispatcher is never called when signature verification fails.

Reply:
removed the test